test/unit/operations/aggregate.test.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const { expect } = require('chai');
+const { AggregateOperation } = require('../../../src/operations/aggregate');
+
+describe('AggregateOperation', function () {
+  const db = 'test';
+
+  describe('#constructor', function () {
```

Comment:
New unit tests to check that the operation gets flagged properly when the aggregation is a write.